components/03-sections/01-accordion/accordion.scss

```diff
@@ -1,95 +1,149 @@
+	border-radius: 0.25rem;
+	display: flex;
+	flex-direction: column;
+	min-width: 0;
+	position: relative;
 }
-#accordion .card-header>.card-link {
-	display: block;
-	padding: 20px 31px;
-	font-size: 21px;
+
+
+#accordion .accordion-item .accordion-button {
 	background-color: $blue;
 	color: $white;
-	position: relative;
+	display: block;
+	font-size: 21px;
 	font-weight: 600;
-	margin-bottom: 0px;
+	margin-bottom: 0;
+	padding: 20px 31px;
+	position: relative;
 }
-#accordion .close-accordion {
-	border: none;
-	padding: 10px 30px;
-	font-size: 32px;
-	font-weight: 300;
-	position: absolute;
+
+
+
+
+
+#accordion .accordion-button:not(.collapsed) {
+	color: $blue;
+	background-color: $grey;
+	border-top: 6px solid $orange-a11y;
+	margin-top: -1px;
+}
+
+#accordion .accordion-button:not(.collapsed)::after {
+	align-items: center;
+	background-color: $blue;
+	bottom: 0;
+	color: $white;
+	display: flex;
+	height: 100%;
+	transform: rotate(0deg);
+	justify-content: center;
 	right: 0;
 	top: 0;
-}
-#accordion .close-accordion:hover {
-	background-color: $orange-a11y;
-	-webkit-transition: all 0.3s ease-in-out;
-	-o-transition: all 0.3s ease-in-out;
-	transition: all 0.3s ease-in-out;
-}
-#accordion .card-body {
-	background-color: $white-b;
-	padding: 15px 32px 94px 32px;
-	position: relative;
-}
-#accordion .card {
-	border: none;
-}
-#accordion .card-header > .card-link {
-    display: block;
-    padding: 20px 31px;
-    font-size: 21px;
-    background-color: $blue;
-    color: $white;
-    position: relative;
-    font-weight: 600;
-    margin-bottom: 0px;
-}
-#accordion .active .card-link {
-    color: $white;
-    background-color: $grey;
-    border-top: 6px solid $orange-a11y;
-}
-#accordion .card-header>.card-link::after {
-	position: absolute;
-	right: 31px;
+	width: 77px;
+}
+
+#accordion .accordion-button:after {
+	background-repeat: no-repeat;
+	background-size: 1.25rem;
+	flex-shrink: 0;
+	height: 1.25rem;
+	margin-left: auto;
+	transition: transform 0.2s ease-in-out;
+	width: 1.25rem;
+	content: "+";
 	font-size: 40px;
-	line-height: 28px;
 	font-weight: 300;
+	line-height: 28px;
+	position: absolute;
+	right: 31px;
 	top: 15px;
 }
-#accordion .card-header>.card-link::after {
-	content: '+';
+
+#accordion .accordion-button:not(.collapsed):after {
+	content: "×";
 }
-#accordion .card-header.active>.card-link::after {
-	content: '\00d7';
+
+#accordion .card-body {
+	background-color: #f6f6f6;
+	padding: 15px 32px 94px;
+	position: relative;
+
 }
-.accordion-body-cntnt {
-	padding-left: 85px;
-	-webkit-box-sizing: content-box;
+
+#accordion .accordion-body-content {
 	box-sizing: content-box;
-	max-width: 648px;
+	max-width: 85%;
+	padding-left: 85px;
 	width: 100%;
 }
-.accordion-body-content h5 {
+
+#accordion .accordion-body-content h5 {
 	color: $orange-a11y;
-	font-weight: 400;
 	font-size: 28px;
+	font-weight: 400;
+	font-family: "open sans", sans-serif;
 }
-.accordion-body-content p {
+
+#accordion .accordion-body-content p {
```

Comment:
Should this be removed according to #301 ?

Reply:
No. otherwise it doesn't pick up the other styles for the accordion paragraph of 
.accordion-body-content p {
    color: #0c2340;
    margin-bottom: 30px;
    margin-top: 30px;
}